### PR TITLE
MAT-7174: Bugfix formik errors not displaying as expected

### PR DESCRIPTION
**Pull request by mcmcphillips**
## MADiE PR

Jira Ticket: [MAT-7174](https://jira.cms.gov/browse/MAT-7174)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included
* [x] No extemporaneous files are included (i.e Complied files or testing results)
* [x] This PR is in to the **correct branch**.
* [x] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [x] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependancies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/components/common/CreateNewLibraryDialog.tsx [hunk 1]
```diff
@@ -142,6 +142,19 @@ const CreateNewLibraryDialog: React.FC<TestProps> = ({
...
 
+  const getFormikErrorMessage = (() => {
+    if (
+      (formik.touched["cqlLibraryName"] || focusedField === "cqlLibraryName") &&
+      formikErrorHandler("cqlLibraryName")
+    ) {
+      return formikErrorHandler("cqlLibraryName");
+    } else {
+      if (formik.values.model === Model.QDM_5_6) {
+        return "Library name must start with an upper case letter, followed by alpha-numeric character(s) and must not contain spaces or other special characters except of underscore for QDM.";
```
Comment (jkotanchik-SB) [inline src/components/common/CreateNewLibraryDialog.tsx]: "except of underscore" reads a little funny. Maybe just "except underscore"?

Comment (mcmcphillips) [reply src/components/common/CreateNewLibraryDialog.tsx]: updated

**Review by jkotanchik-SB — APPROVED** [review]
